var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -13,32 +13,42 @@ class Raja(CachedCMakePackage, CudaPackage, ROCmPackage):
     """RAJA Parallel Framework."""
 
     homepage = "https://software.llnl.gov/RAJA/"
+    url = "https://github.com/LLNL/RAJA/releases/download/v2022.03.0/RAJA-v2022.03.0.tar.gz"
```

Comment:
You could remove this property and add something along the lines of the following as the first method (after the dependencies, so around line 103):

```
    def url_for_version(self, version):
        vers = f"v{version}" if version > Version("0.8") else str(version)
        return f"https://github.com/LLNL/RAJA/releases/download/v{version}/RAJA-{vers}.tar.gz"
```